documentation/source/newsfragments/1286.doc.rst

```diff
@@ -0,0 +1 @@
+New example ``matrix_multiplier``.
```

Comment:
#1286 isn't merged, so let's not do this yet. Can you instead add one for #1255, which is merged and release-note worthy?